DXRModules/DeusEx/Classes/DXREntranceRando.uc

```diff
@@ -519,14 +570,40 @@ function GenerateConnections(int missionNum)
     numConns = 0;// vanilla on failure
 }
 
+function BuildCache()
```

Comment:
building the cache costs quite a few loop iterations, but it seems to greatly improve the worst-case scenarios